store/siren-action-behavior.html

```diff
@@ -123,9 +129,9 @@
 					});
 				})
 				.catch(function(reason) {
-					self.fire('d2l-siren-entity-save-error', { error: new Error(reason.message) });
-					throw reason;
-				});
+					self.fire('d2l-siren-entity-save-error', reason);
```

Comment:
Just an FYI - we have code in d2l-rubrics that looks for the `error` property of the event and I think just includes it as a hidden field when it shows the event. 

Also I think we should just throw the original reason as before, otherwise there is a possibility we slice off other information that may have been included in the original error reason.

For similar reasons, I don't think we need to rewrap the error object we add to the event.
We could just do 
`self.fire('d2l-siren-entity-save-error', { error: reason });`

I don't particularly care if we just throw the original parsed JSON object or we wrap it in another object with the JSON object and the formatted errMsg.